code/modules/medical/genetics/bioEffects/speech.dm

```diff
@@ -150,6 +150,29 @@
 		message = finnishify(message)
 		return message
 
+/datum/bioEffect/speech/german
+	name = "Frontal Gyrus Alteration Type-DE"
+	desc = "Forces the language center of the subject's brain to construct scentences in a German manner."
```

Comment:
typo typo typo typo typo typo

Reply:
ach- sorry. Dysgraphia moment. I always spell that wrong. Fixing now